packages/filesystem/src/node/download/file-download-cache.ts

```diff
@@ -70,11 +70,11 @@ export class FileDownloadCache {
     }
 
     protected deleteRecursively(pathToDelete: string): void {
-        rimraf(pathToDelete, error => {
-            if (error) {
-                this.logger.warn(`An error occurred while deleting the temporary data from the disk. Cannot clean up: ${pathToDelete}.`, error);
-            }
-        });
+        try {
+            rimraf(pathToDelete);
+        } catch (error) {
+            this.logger.warn(`An error occurred while deleting the temporary data from the disk. Cannot clean up: ${pathToDelete}.`, error);
+        }
```

Comment:
This try-catch will not actually catch anything as the `rimraf` is not awaited. After triggering `rimraf` the try is already completed and any thrown error will not be caught.

To restore the old behavior of error logging, the error handling must be part of the catch, i.e.
```suggestion
        rimraf(pathToDelete).catch(error => {
            this.logger.warn(`An error occurred while deleting the temporary data from the disk. Cannot clean up: ${pathToDelete}.`, error);
        });
```